followthemoney/schema/Sanction.yaml

```diff
@@ -18,7 +18,7 @@ Sanction:
     entity:
       label: "Entity"
       reverse:
-        name: sanctionEntity
+        name: sanctions
```

Comment:
Not within the data itself, since it's a sort of virtual reverse property that doesn't ever get stored in the data. The only issue might be UI deep links to the "sanctions" tab of a person, which would now go to the person base page. I'd be very surprised if any links like that were in use.